src/app/modules/compare/layout/compare-layout.component.ts

```diff
@@ -62,7 +62,7 @@ export class CompareLayoutComponent implements OnInit, OnDestroy {
         this.subs.push(this.productService.getProducts(null, null, null, null, null, null, conditions).pipe(map(res => res.Products)).subscribe(products => {
           const tableProducts = products.filter(product => newIdentifiers.includes(product[this.identifier]));
           this.products = tableProducts;
-          this.batchSelectionService.setSelectedProducts(tableProducts);
+          this.BatchSelectionService.setSelectedProducts(tableProducts);
```

Comment:
Please keep it back to batchSelectionService, service names should follow camel case

Reply:
Addressed the comment